v63003/outref/gtm4212.txt

```diff
@@ -0,0 +1,20 @@
+# Backing up Default to length 230 path (length of temp file is 24, so total path is 254)
+##TEST_AWKDB file ##TEST_PATH##/mumps.dat backed up in file ##TEST_PATH##[a/]*mumps.dat
+##TEST_AWKTransactions up to 0x................ are backed up.
+
+
+BACKUP COMPLETED.
+
+# Backing up Default to length 231 path (length of temp file is 24, so total path is 255)
+##TEST_AWK%YDB-E-FILEPARSE, Error parsing file specification: ##TEST_PATH##[a/]*DEFAULT_........_XXXXXX
+%YDB-E-FILENAMETOOLONG, File name too long
+# Backing up a Default to length <=230 path (<=254 including temp file)
+##TEST_AWKDB file ##TEST_PATH##/mumps.dat backed up in file ##TEST_PATH##[a/]*mumps.dat
+##TEST_AWKTransactions up to 0x................ are backed up.
+
+
+BACKUP COMPLETED.
+
+# Backing up a Default to length >=231 path (>=255 including temp file)
```

Comment:
```>=231``` path also has some lower limit right? If so do indicate that (i.e. <= 241 or so).